PROVISIONING-METAL.md

```diff
@@ -0,0 +1,169 @@
+# Povisioning Bottlerocket on metal
+
+This guide will describe what is needed to properly provision Bottlerocket on bare metal.
+Provisioning Bottlerocket on metal is different than provisioning other general-purpose distros.
+Since Bottlerocket has a `dm-verity`-checked boot and root partition, and is immutable at runtime, we cannot simply provision an image and directly write configuration files.
+Bottlerocket requires a few files to be generated and written to disk at provisioning time in order to boot properly; these files are described below.
+
+> For more information about the hardware that Bottlerocket is currently tested on, see [Hardware](HARDWARE.md).
+
+## Prerequisites
+
+* Bottlerocket metal image
+  * [`tuftool`](https://github.com/awslabs/tough/blob/develop/tuftool/README.md) is required to fetch the image
+* [User data TOML file](#user-data): `user-data.toml`
+* [Network configuration TOML file](#network-interface-configuration): `net.toml`
+* (Optional) [Boot configuration file](#boot-configuration): `bootconfig.data`
```

Comment:
"Prerequisites" might not be the right word; that tells me I need these things before continuing, when the purpose of the guide is to tell me how to get them.

The high-level steps below might work as a replacement for this section.